pom.xml

```diff
@@ -384,12 +384,12 @@
               <exclude>**/*IntegrationTest.java</exclude>
             </excludes>
             <argLine>-Xmx1024m -Dfile.encoding=UTF-8</argLine>
-            <systemProperties>
-              <property>
-                <name>apple.awt.UIElement</name>
-                <value>true</value>
-              </property>
-            </systemProperties>
+            <systemPropertyVariables>
+              <apple.awt.UIElement>true</apple.awt.UIElement>
```

Comment:
Nothing really. `<systemProperties>` is deprecated (see https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#systemProperties) so I just replaced it with `<systemPropertyVariables>`.